specification/devcenter/DevCenter/DevBox/models.tsp

```diff
@@ -228,6 +337,10 @@ Overall health status of the Pool. Indicates whether or not the Pool is
 available to create Dev Boxes.
 """)
   healthStatus: PoolHealthStatus;
+
+  @doc("Display name of the pool.")
+  @added(APIVersions.v2024_02_01)
+  displayName?: string;
```

Comment:
Yeah its basically just the friendly name of the pool that a user can set, I believe it's what the UX will show rather than the actual poolName in the resource ID of the pool (although not 100% sure on that last part)